datasets/pl/wanted/crawler.py

```diff
@@ -0,0 +1,160 @@
+                citizenship_1 = " ".join(split_citizenships[:i])
+                citizenship_2 = " ".join(split_citizenships[i:])
+                if countrynames.to_code(
+                    citizenship_1, fuzzy=True
+                ) and countrynames.to_code(citizenship_2, fuzzy=True):
+                    return [citizenship_1, citizenship_2]
+
+    # Sometimes, there is a full spelling of the country in parens
+    if alt_country_match:
+        alt_country = alt_country_match.groupdict().get("alt_country")
+        if countrynames.to_code(alt_country, fuzzy=True):
+            return alt_country
+
+    # If nothing worked, we just return the original value
+    return citizenship
+
+
+def crawl_person(context: Context, url: str):
+    context.log.debug(f"Crawling person page {url}")
+
+    person = context.make("Person")
+    # TODO(Leon Handreke): Do we use the r12323456 in the URL that looks like a record ID?
+
+    # There is an ID prefixed by r in the URL, which seems to refer to the record. The name component in the URL
+    # can be changed at will.
+    # URLs look like this: https://poszukiwani.policja.gov.pl/pos/form/r922391307881,OCHNIO-KRZYSZTOF.html
+    url_match = re.search(r"/pos/form/(?P<rid>r\d+),.+.html", url)
+    assert (
+        url_match and "rid" in url_match.groupdict()
+    ), "Could not extract r-ID from malformed person URL"
+    rid = url_match.groupdict()["rid"]
+    person.id = context.make_slug(rid)
+
+    doc = context.fetch_html(url, cache_days=7)
+    # Extract details using XPath based on the provided HTML structure
+    details = {
+        "full_name": "//div[@class='head']/h2/text()",
+        "middle_name": "//p[contains(text(), 'Data urodzenia:')]/strong/text()",
+        "father_name": "//p[contains(text(), 'Imię ojca:')]/strong/text()",
+        "mother_name": "//p[contains(text(), 'Imię matki:')]/strong/text()",
+        "mother_maiden_name": "//p[contains(text(), 'Nazwisko panieńskie matki:')]/strong/text()",
+        "gender": "//p[contains(text(), 'Płeć:')]/strong/text()",
+        "place_of_birth": "//p[contains(text(), 'Miejsce urodzenia:')]/strong/text()",
+        "date_of_birth": "//p[contains(text(), 'Data urodzenia:')]/strong/text()",
+        "alias": "//p[contains(text(), 'Pseudonim:')]/strong/text()",
+        "citizenship": "//p[contains(text(), 'Obywatelstwo:')]/strong/text()",
+        "height": "//p[contains(text(), 'Wzrost:')]/strong/text()",
+        "eye_color": "//p[contains(text(), 'Kolor oczu:')]/strong/text()",
+    }
+    info = dict()
+    for key, xpath in details.items():
+        q = doc.xpath(xpath)
+        if q:
+            text = q[0].strip()
+            if text != "-":
+                info[key] = text
+
+    person.add("sourceUrl", url)
+    person.add("topics", "crime")
+    person.add("topics", "wanted")
+    # TODO(Leon Handreke): What does country mean in that context? The za_wanted crawler also does this.
+    person.add("country", "pl")
```

Comment:
we use country when there's some other association with a country, e.g. personally conducting business there, or in this case being wanted for a crime suggests some connection with the country. I think it can be useful for disambiguation of names.